src/main/java/frc/robot/Constants.java

```diff
@@ -67,11 +67,11 @@
     public static final int STAGE_3_DISTANCE = 10;
     public static final int STAGE_4_DISTANCE = 5;
     
-    public static final double STAGE_1_SPEED = 0.8;
-    public static final double STAGE_2_SPEED = 0.5;
-    public static final double STAGE_3_SPEED = 0.3;
-    public static final double STAGE_4_SPEED = 0.1;
-    public static final double STAGE_5_SPEED = 0.05;
+    public static final double STAGE_1_SPEED = 1.0;
+    public static final double STAGE_2_SPEED = 0.8;
+    public static final double STAGE_3_SPEED = 0.5;
+    public static final double STAGE_4_SPEED = 0.3;
+    public static final double STAGE_5_SPEED = 0.1;
 
```

Comment:
changes how fast limelight moves the robot, low priority but worth testing new numbers if there's enough time.